libkbfs/folder_block_ops.go

```diff
@@ -1344,6 +1344,20 @@ func (fbo *folderBlockOps) writeDataLocked(
 			newb := fblock.IPtrs[len(fblock.IPtrs)-1]
 			copy(fblock.IPtrs[indexInParent+2:], fblock.IPtrs[indexInParent+1:])
 			fblock.IPtrs[indexInParent+1] = newb
+			if oldSize == de.Size {
```

Comment:
may be worth renaming `oldSize` to `oldSizeWithoutHoles` or something, just in case someone is tempted to use it for something else in this (big) function.